arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -135,27 +137,24 @@ bool IdentityAnalyzer::reset(irs::string_ref const& data) {
 arangodb::aql::AqlValue aqlFnTokens(arangodb::aql::ExpressionContext* expressionContext,
                                     arangodb::transaction::Methods* trx,
                                     arangodb::aql::VPackFunctionParameters const& args) {
-  if (2 != args.size() || !args[0].isString() || !args[1].isString()) {
-    LOG_TOPIC("740fd", WARN, arangodb::iresearch::TOPIC)
-        << "invalid arguments passed while computing result for function "
-           "'TOKENS'";
-    TRI_set_errno(TRI_ERROR_BAD_PARAMETER);
+  std::string const while_tokens =
```

Comment:
let's avoid allocating strings for good case

Reply:
I can pull it out the function and make it static.